### ISSUE TITLE
Cannot build initramfs

### ISSUE THREAD

**Issue by dvicory** [midzelis/zquickinit#8]
I'm trying to build a customized version of ZQuickInit but running into an issue in several different build environments. On macOS, Proxmox 8/Debian 12, and [the GitHub workflow](https://github.com/dvicory/zquickinit/actions/runs/10031125064/job/27721220309) (after forking and change references to my user), I get the following error when trying to build initramfs.

```
$ ./zquickinit.sh initramfs --no-ask

Launching docker...
bind-mount: zquickinit.sh (readonly)
bind-mount: /root/zquickinit/recipes to /input (readonly)
bind-mount: /root/zquickinit/output to /output (read-write)


  ╔══════════════════════════════════════════════════╗
  ║       Welcome to ZQuickInit make initramfs       ║
  ╚══════════════════════════════════════════════════╝

ZQuickInit (https://github.com/midzelis/zquickinit) commit hash: 8052aa4 (8052aa4982587720554da524a51f298bd238d037)
ZBM (https://github.com/zbm-dev/zfsbootmenu) commit hash: f23fc69 (f23fc698c42220593a011cf6b58a0220452225f0)
find: '/lib/modules': No such file or directory
mkinitcpio: option '--kernel' requires an argument
```

I really love that you solved several issues that ZFSBootMenu has while making it much easier to get started, so I'm hoping the fix for this isn't too involved.

**Comment by midzelis** [comment]
Hey - give it another go. I've made a bunch of improvements.

**Comment by midzelis** [comment]
Fixed via #11
